build.sbt

```diff
@@ -24,6 +24,7 @@ lazy val commonSettings = Seq(
   organizationHomepage := Some(url("https://cognite.com")),
   version := "1.4.63",
   crossScalaVersions := supportedScalaVersions,
+  scalaVersion := scala213, // default to Scala 2.13
```

Comment:
It's better to default to 2.12, as the IDE will pick up this setting, and code that is valid in 2.13 may not work on 2.12. It's better to not get recommendations from the IDE that may depend on 2.13.

Reply:
Ok, good point. I defaulted to 2.13, because tests were mostly broken on that version, but you are right. btw, do you know when we can drop 2.12? Does anyone except Jetfire depend on that?